src/components/app/Home.js

```diff
@@ -337,10 +337,12 @@ class Home extends React.PureComponent<HomeProps, HomeState> {
       >
         <section className="homeSection">
           <h1 className="homeTitle">
-            <span className="homeTitleText">perf.html</span>
-            <span className="homeTitleSubtext">
-              {' '}
-              &mdash; Web app for Firefox performance analysis
+            <span className="homeTitleSlogan">
```

Comment:
I wraped .homeTitleText and .homeTitleSubtext in .homeTitleSlogan to easier styling in small screen, where github icon doesn't fit into the line.